plugins/inputs/mysql/v2/convert.go

```diff
@@ -0,0 +1,94 @@
+package v2
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"strconv"
+)
+
+type ConversionFunc func(value sql.RawBytes) (interface{}, error)
+
+func ParseInt(value sql.RawBytes) (interface{}, error) {
+	v, err := strconv.ParseInt(string(value), 10, 64)
+
+	// Ignore ErrRange.  When this error is set the returned value is "the
+	// maximum magnitude integer of the appropriate bitSize and sign."
+	if err, ok := err.(*strconv.NumError); ok && err.Err == strconv.ErrRange {
+		return v, nil
```

Comment:
Are we expecting ints bigger than int64 or is this here to be defensive?

Reply:
Expecting larger integers, I believe this is actually the cause of the 5529 bug.  In the old strategy we tried to parse as an integer and then fell back to a float64, causing the conflict.  In theory we also have a unsigned integer type that we could work with, but there isn't a clean way to migrate to it.